<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Pixel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PIXEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_RoteHorizLinie">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Rote horizontale Linie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GrueneVertikLinie">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Grüne vertikale Linie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Kreuz">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Kreuz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SchraegeLinie">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Schräge Linie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AllesRot">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ganze Bildfläche rot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Quadrat">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Quadrat Special</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Farbverlauf">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Farbverlauf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Kreis">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Kreis Special</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
